pkg/kyc/authorization.go

```diff
@@ -175,6 +177,23 @@ func (c *authorizationComponentMW) ValidateUser(ctx context.Context, realmName s
 	return c.next.ValidateUser(ctx, realmName, userID, user, consentCode)
 }
 
+/********************* (BEGIN) Temporary basic identity (TO BE REMOVED WHEN MULTI-ACCREDITATION WILL BE IMPLEMENTED) *********************/
+func (c *authorizationComponentMW) ValidateUserBasicID(ctx context.Context, userID string, user apikyc.UserRepresentation) error {
+	var action = kycValidateUserBasicID.String()
+
+	// For this method, there is no target realm provided
+	// as parameter, so we pick the current realm of the user.
+	var targetRealm = ctx.Value(cs.CtContextRealm).(string)
+
+	if err := c.authManager.CheckAuthorizationOnTargetRealm(ctx, action, targetRealm); err != nil {
```

Comment:
I think the authorization should be OnTargetUser

Reply:
The authorization is from scope realm (and this is expected). So it should be fine with CheckAuthorizationOnTargetRealm